src/connector/src/sink/iceberg/mod.rs

```diff
@@ -414,11 +418,14 @@ impl Sink for IcebergSink {
         let catalog = self.config.create_catalog().await?;
         let table = self.create_and_validate_table().await?;
         let partition_type = table.current_partition_type()?;
+        let (commit_tx, finish_tx) = spawn_compaction_client(&self.config)?;
```

Comment:
Is it expected to spawn this compact client for all iceberg sink?

Reply:
Good catch